build.gradle

```diff
@@ -672,6 +672,7 @@ project(':streams') {
     testCompile project(':core')
     testCompile project(':core').sourceSets.test.output
     testCompile libs.junit
+    testCompile libs.assertj
```

Comment:
@ijuma the hamcrest included with junit unfortunately doesn't have the "contains" Matcher included.